src/main/java/com/internetitem/logback/elasticsearch/writer/SafeWriter.java

```diff
@@ -1,12 +1,14 @@
-package com.internetitem.logback.elasticsearch.writer;
-
-import java.io.IOException;
-
-public interface SafeWriter {
-
-	void write(char[] cbuf, int off, int len);
-
-	void sendData() throws IOException;
-
-	boolean hasPendingData();
-}
+package com.internetitem.logback.elasticsearch.writer;
+
+import java.io.IOException;
+import java.security.KeyManagementException;
+import java.security.NoSuchAlgorithmException;
+
+public interface SafeWriter {
+
+	void write(char[] cbuf, int off, int len);
+
+	void sendData() throws IOException, NoSuchAlgorithmException, KeyManagementException;

```

Comment:
I think it would be better to either declare that `sendDate()` might throw `Exception`, or catch and wrap the SSL ones in `IOException`s (since after all, the end result is a problem with IO)